core/messages/da.json

```diff
@@ -41,8 +41,24 @@
             "multipleAddresses": "Gemme flere forsendelsesadresser",
             "ordersHistory": "Få adgang til din ordrehistorik",
             "ordersTracking": "Spore nye ordrer",
-            "wishlists": "Save items to your Wishlist",
+            "wishlists": "Gemme varer til din ønskeliste",
             "createLink": "Opret konto "
+        },
+        "ForgotPassword": {
+            "title": "Forgot password",
+            "heading": "Forgot password",
```

Comment:
Added after most of these phrases got translated. Going to merge even though this is off.